jck/README.md

```diff
@@ -10,13 +23,30 @@
 1. Put unarchived jck test materials (jck8b or jck9) into an empty folder, for example:
 * `/jck/jck8b/` and `/jck/jck9`
 
-2. Export `JCK_ROOT=/jck` as an environment variable or pass it in makefile when run make command
+2. Export `JCK_ROOT=/jck` as an environment variable or pass it in makefile when run make commands
 
-3. Export `JAVA_HOME=<your_JDK_root>` as an environment variable
+3. Export `JCK_VERSION=<your_jck_version>` as an environment variable or pass it in makefile when run make commands. For example `export JCK_VERSION=jck8b` 
```

Comment:
Typo?.. Could be phrased as "when run as a make command".

Reply:
Thanks for pointing it out Mesbah, updated with newer chageset